CMakeLists.txt

```diff
@@ -34,42 +28,20 @@ if(POLICY CMP0012)
   cmake_policy(SET CMP0012 NEW)
 endif()
 
-find_package(OpenMP)
+if(OPENMP)
+  find_package(OpenMP REQUIRED COMPONENTS Fortran)
+endif()
+
 find_package(LAPACK)
 find_package(Jasper REQUIRED)
 find_package(PNG REQUIRED)
 find_package(ZLIB REQUIRED)
```

Comment:
Most probably none of the utilities depend on these three (Japser, PNG and ZLIB) packages directly, but I didn't check.

Reply:
I'm not sure about this. Can we approve it and I'll take a look at what is actually required?